test/com/sun/jna/VarArgsTest.java

```diff
@@ -43,6 +43,7 @@
         public int addVarArgs(String fmt, Number... args);
         public String returnStringVarArgs(String fmt, Object... args);
         public void modifyStructureVarArgs(String fmt, Object arg1, Object... args);
+        public String returnStringVarArgs(String... args);
```

Comment:
Varargs with no fixed arguments aren't possible in C (though they may be in other languages).  In C you'd have to directly access the stack pointer to find the address of arguments, which may be compiler or processor-dependent.